README.md

```diff
@@ -1,56 +1,53 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] OPTIONAL:  In your pull request description, your hard work styling is not showing up.  This is because you are missing a required space between each `-` and the item content.  I suggest you add those spaces and see for yourself the results!

Example of what can be done: 

![image](https://user-images.githubusercontent.com/57421823/154230953-c3f01c34-7d9d-4086-97a7-629f29d4143a.png)

Result:

![image](https://user-images.githubusercontent.com/57421823/154231010-dc89c307-039b-4a61-89fb-c8953558d5cb.png)

I hope this is helpful 💪